copier.yaml

```diff
@@ -36,10 +41,24 @@ _tasks:
     -a use_pre_commit "no"'
 
   - "echo '==== 4/4 - Finalizing setup ===='"
-  - '"{{ python_path if python_path else _copier_python }}" post_init.py "./projects/{{ project_name }}-contracts" "./projects/{{ project_name }}-frontend"'
-  # _copier_python above is used for backwards compatibility with versions < v1.11.3 of the algokit cli
+  # Execute OS-specific file copying script (with self-destruction)
+  - command: |
+      chmod +x ./copy_files.sh
+      ./copy_files.sh "{{ project_name }}"
+    when: "{{ _copier_conf.os in ['linux', 'macos'] }}"
+  - command: |
+      powershell -File ./copy_files.ps1 -project_name "{{ project_name }}"
+    when: "{{ _copier_conf.os == 'windows' }}"
+
+_message_after_copy: >
+  ✅ Fullstack project successfully created!
+
+  You can now:
+  - cd into projects/{{ project_name }}-contracts and projects/{{ project_name }}-frontend to work on your project
```

Comment:
Not sure this is needed to be honest. VS Code is automatically launched and people can explore easily.